include/revng/mlir/Dialect/Clift/IR/CliftTypes.td

```diff
@@ -452,6 +452,10 @@ def Clift_StructType : Clift_ClassTypeBase<"Struct", "struct"> {
       return $_get($_ctxt, StructAttr::get($_ctxt, Handle), /*IsConst=*/false);
     }]>,
 
+    TypeBuilder<(ins "mlir::clift::StructAttr":$Definition), [{
+      return $_get($_ctxt, Definition, /*IsConst=*/false);
+    }]>,
+
```

Comment:
I think this builder belongs to the commit that introduces `ClassDefinition` in place of `ClassDefinitionAttr`.

Same for the builder for unions with the same API.